src/textOnlyPackages/src/microsoft.netcore.platforms/1.0.1/Microsoft.NETCore.Platforms.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netcore.platforms/1.0.1/</OutputPath>
+    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netcore.platforms/1.0.1/</BaseIntermediateOutputPath>
```

Comment:
Use project-agnostic csproj to avoid having to review these proj files? (Maybe just in the future--I didn't find any particular problem reviewing the ones in this PR.)

I.e. this one:

https://github.com/dotnet/source-build-reference-packages/blob/fe6afa24c69cc135200bbe1f12a90a11a450dcb0/src/textOnlyPackages/src/microsoft.netcore.dotnetapphost/2.0.0/Microsoft.NETCore.DotNetAppHost.csproj#L3-L10

Reply:
Oh, nice, I didn't realize we had this.  Yes, I'll definitely do that next time.